app/config/sources.inc.php

```diff
@@ -522,6 +522,11 @@
         'priority'          => 1,
         'supported_locales' => $mozillaorg,
     ],
+    'privacy/faq.lang' => [
+        'priority'          => 2,
+        'deadline'          => '2018-07-30',
+        'supported_locales' => $mozillaorg,
```

Comment:
I don't think this is correct. The privacy main page (privacy/index.lang) is for a much smaller set of locales.

Reply:
There is a page called privacy/principles.lang which supports all mozorg locales, so I thought this should be at the same level.

Reply:
Have you tried opening it? It redirects me to https://www-dev.allizom.org/internet-health/privacy-security/, which makes me think it was removed at some point.

Reply:
> Have you tried opening it? It redirects me to https://www-dev.allizom.org/internet-health/privacy-security/, which makes me think it was removed at some point.

Never mind, that's not the principles page.

Reply:
You mean this one: https://www.mozilla.org/it/privacy/principles/

Reply:
It's not discoverable without the index (I've updated the other comment in the meantime).

Maybe it makes sense to open the other page to all locales, with the caveat of not being able to translate policies. Anyhow, your call ;-)

Reply:
I hate to make decisions at this time of the day!! 
How discoverable is the "principles". I have a feeling the FAQ page could be linked from other places down the road. Can we make it an opt-in page?

Reply:
I will leave this one as is, and decide on the Index page after talking to legal. It doesn't look good for sure.

Reply:
@flodolo ok to proceed otherwise?